### PR TITLE
implement the p11mod logout function

### PR DESCRIPTION
**Pull request by KingAnointing**

### CODE REVIEW

**Comment by JeremyRand** [comment]
The Cirrus fails are likely to be spurious, triggered a rebuild.

File: p11mod/p11mod.go [hunk 1]
```diff
@@ -391,9 +395,16 @@ func (ll *llBackend) Login(sh pkcs11.SessionHandle, userType uint, pin string) e
...
-	// TODO
-	log.Println("p11mod Logout: not implemented")
-	return nil
+	session, err := ll.getSessionByHandle(sh)
+	if err != nil {
+		return err
+	}
+
+	if trace {
+		log.Println("p11mod Logout user ")
```
Comment (JeremyRand) [inline p11mod/p11mod.go:404]: Please remove the ` user ` suffix; it's unnecessary.

**Comment by JeremyRand** [comment]
ACK 4ec55faf182d3b1e94160b064dee02966c7d2d7b modulo the suffix in the trace message.

File: p11mod/p11mod.go (hunk 1 above)
Comment (KingAnointing) [reply p11mod/p11mod.go:404]: Sorry, I did not notice I added the user suffix 
I would do that asap :)

**Comment by JeremyRand** [comment]
ACK f55988e08b6db4324065552a515c77c91600cdde, modulo the trailing space.

File: p11mod/p11mod.go [hunk 2]
```diff
@@ -391,9 +395,16 @@ func (ll *llBackend) Login(sh pkcs11.SessionHandle, userType uint, pin string) e
...
-	// TODO
-	log.Println("p11mod Logout: not implemented")
-	return nil
+	session, err := ll.getSessionByHandle(sh)
+	if err != nil {
+		return err
+	}
+
+	if trace {
+		log.Println("p11mod Logout ")
```
[the remark this answers is not in the document]
Comment (KingAnointing) [reply p11mod/p11mod.go]: I removed the trail

**Comment by JeremyRand** [comment]
ACK 8614c4a4b25f70aeb6b18fec9378c6911e9c0dc6 if Cirrus passes.

**Comment by JeremyRand** [comment]
ACK 8614c4a4b25f70aeb6b18fec9378c6911e9c0dc6

**Pull request merged**